src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
```

Comment:
@AArnott that questions bugs me also. I tend to prefer the explicit value as it mostly will be optimized if it is the same as the default one but increases readability by stating that it was something in purpose instead of something that was forgotten or overlooked.